tests/unit/string_util_test.cpp

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#include <gtest/gtest.h>
+
+using namespace testing;
+
+extern "C" {
+   #include <tilck/common/string_util.h>
+   char *tilck_strstr(const char *haystack, const char *needle);
+   char *tilck_strncpy(char *dest, const char *src, size_t n);
```

Comment:
The whole point of putting those `tilck_*` declarations inside the header is to remove them from this file, because it's unsafe to put them here (there are no compile-time checks validating that the signature matches).

Reply:
sorry, again.